src/main/java/david/ui/Ui.java

```diff
@@ -0,0 +1,142 @@
+package david.ui;
+
+import java.util.Scanner;
+
+import david.task.Task;
+import david.task.TaskList;
+
+/**
+ * Ui class for I/O operations
+ */
+public class Ui {
+    private static final String INTRO =
+            "____________________________________________________________\n"
```

Comment:
Perhaps you could consider initialising a constant 'LINE' since this line is used a couple of times in your code